electrum/gui/qt/__init__.py

```diff
@@ -303,13 +302,15 @@ def start_new_window(self, path, uri, *, app_is_starting=False):
 
     def _start_wizard_to_select_or_create_wallet(self, path) -> Optional[Abstract_Wallet]:
         wizard = InstallWizard(self.config, self.app, self.plugins)
+        if_run_notification = False
```

Comment:
Change this name just to `run_notification`

Reply:
done